docs/source/release-notes.html.md

```diff
@@ -7,6 +7,19 @@ weight: 8
 
 If you have any questions or comments about these notes, please contact DfE via Slack or email.
 
+## 29 August 2023
```

Comment:
Could you amend this date to maybe today/tomorrow depending on when this is set to be merged deployed?